extensions/2.0/Khronos/KHR_gaussian_splatting/README.md

```diff
@@ -0,0 +1,374 @@
+
+Renderers are free to ignore any values they do not recognize, but are encouraged to follow the non-normative list below.
+
+#### Known Shapes
+
+*This section is non-normative and not comprehensive. It may change over time.*
+
+ - `ellipsoid` _(Default Value)_
+ - `triangle`
+ - `quad`
+
+### Rendering Hints
+
+This extension provides a `hints` property that contains sub-properties that may help renderers understand how best to render the Gaussians to the screen. This property and all of it's sub-properties are optional, and renderers can choose to ignore them.
+
+#### Projection
+
+The `projection` property is an optional hint that specifies how the Gaussians should be projected into the image. This is typically provided by the training process for the splats. This is a freeform string field to allow new projections to be specified as they become available. The default value is `perspective`.
+
+Renderers are free to ignore any values they do not recognize, but are encouraged to follow the non-normative list below.
+
+##### Known Projection Methods
+
+*This section is non-normative and not comprehensive. It may change over time.*
+
+ - `perspective` _(Default Value)_: The typical 3D perspective projection based on scene depth.
+ - `orthographic`: A orthogonal projection of splats into a scene to preserve shape and scale and reduce distortion.
+
+#### Sorting Method
+
+The `sortingMethod` property is an optional hint that specifies how the Gaussians should be sorted during the rendering process. This typically is provided by the training process for the splats. This is a freeform string field to allow new sorting methods to be specified as they become available. The default value is `cameraDistance`.
+
+Renderers are free to ignore any values they do not recognize, but are encouraged to follow the non-normative list below.
+
+##### Known Sorting Methods
+
+*This section is non-normative and not comprehensive. It may change over time.*
+
+ - `cameraDistance` _(Default Value)_: The distance between the center of the splat and the position of the camera.
+ - `zDepth`: The projected z-depth in the camera projection.
+
+## Attributes
+
+Each 3D Gaussian splat has the following attributes. At minimum the attributes must contain `POSITION`, `COLOR_0`, `KHR_gaussian_splatting:ROTATION`, and `KHR_gaussian_splatting:SCALE`. `KHR_gaussian_splatting:SH_DEGREE_ℓ_COEF_n` attributes hold the spherical harmonics data and are not required. `POSITION` and `COLOR_0` are defined by the base glTF specification. If higher degrees of Spherical Harmonics are used then lower degrees are required implicitly.
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type | Required | Notes |
+| --- | --- | --- | --- | --- | --- |
+| Rotation | KHR_gaussian_splatting:ROTATION | VEC4 | float | yes | Rotation is a quaternion. (xyzw) |
```

Comment:
Can we permit quantized (normalized integer) rotations here? Similarly to animation rotations, or mesh instancing, this would allow to reduce GPU memory impact and allow for higher degree of compression when using meshopt.